components/Header.js

```diff
@@ -60,13 +60,15 @@ export default function Header({ translationPath }) {
             </a>
           </WebsiteLink>
           <button
+            id="menuButton"
             className="lg:hidden text-blue pr-8 w-16"
             onClick={() => setMenuOpen(true)}
           >
             <MenuIcon />
           </button>
           {/* -------- Mobile Navigation -------- */}
           <div
+            id="menuListMobile"
```

Comment:
How about using `data-cy` attributes, like [documented here](https://docs.cypress.io/guides/references/best-practices#Selecting-Elements)? I like the clear separation between JS/CSS and Cypress.

Reply:
Totally agree